mautic/mautic
# Fix 10455: Allows to reopen the code edit modal after grapesjs has been closed

closes #10455

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | yes
| Deprecations?                          |  no
| BC breaks? (use the c.x branch)        | no
| Automated tests included?              | N/A
| Issue(s) addressed                     |  see new pr for 4.1: https://github.com/mautic/mautic/pull/10689

#### Description:

This fixes an issue where the code editor modal did not open **again** when the grapesjs builder was closed.

#### Steps to test this PR:

1. Load up [this PR](https://mautibox.com)
2. See steps in the [issue](https://github.com/mautic/mautic/issues/10455)


<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10564"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
